.ui/library/flRow2LineButton.cpx

```diff
@@ -18,8 +18,8 @@
 			"type": "FlexLayout",
 			"userProps": {
 				"flexProps": {},
-				"left": 11.76,
-				"top": 123.53
+				"left": 4,
```

Comment:
Could you remove user props from this line? @suleymanbasbug

Reply:
OK.